vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -6,20 +6,19 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
-          "1.24.10-hotfix.20230509-2",
           "1.24.10-hotfix.20230509.1",
```

Comment:
It seems like that this file is not updated in https://github.com/Azure/AgentBaker/commit/3da23e3766b81e63984e8cf4b5178b6f92a8b37a cc @mainred and @andyzhangx

Reply:
I think that should be ok since https://github.com/Azure/AgentBaker/pull/3217 is only for hotfixing kubelet, this change hotfixes the kube-proxy.